frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     });
     return (
       <>
-        {!loading && (
-          <div className="co-m-pane__filter-bar co-m-pane__filter-bar--with-help-text">
```

Comment:
`co-m-pane__filter-bar--with-help-text` can be removed from the SCSS.

Reply:
removed these classes from SCSS